### PR TITLE
Migrate to Cosmos-SDK 0.39.0-rc

### PR DESCRIPTION
**Pull request by alessio**
Hi!

This patch is meant to facilitate your migration to Cosmos-SDK's new release series `v0.39` (codename **Launchpad**).
**Launchpad** releases will be maintained according to a fairly strict [stable release policy](https://github.com/cosmos/cosmos-sdk/blob/master/STABLE_RELEASES.md#stable-release-policy). More importantly, all `v0.38` releases will cease to receive security vulnerability patches and bug fixes as soon as `v0.39.0` is published. Therefore, it's highly recommended to migrate your code as soon as possible.

Thanks for considering.

### CODE REVIEW

**Comment by boz** [comment]
Thanks so much @alessio!

Question: can we use tendermint's `v3` blockchain reactor with this?

**Comment by jackzampolin** [comment]
cc @tessr

File: go.mod [hunk 1]
```diff
@@ -6,7 +6,7 @@ require (
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/boz/go-lifecycle v0.1.1-0.20190620234137-5139c86739b8
 	github.com/caarlos0/env v3.3.0+incompatible
-	github.com/cosmos/cosmos-sdk v0.38.5
```
Comment (jackzampolin) [inline go.mod:9]: So just a quick question here, is this a tagged release? What is the diff between 0.38.5 and 0.38.6-whatever?

Comment (alessio) [reply go.mod:9]: The commit `a0fb47f5fd822889ce6b23fb4de884c76a4273d9` lives in the `launchpad/release/v0.39` (it's an RC branch, see https://github.com/cosmos/cosmos-sdk/pull/6707). The odd `0.38.6-` prefix is due to how `go mod` interprets versioning numbers (sort of mix between a semantic versioning-compliant format a `git describe`'s output`. However, I've just tagged the first 0.39.0 release candidate, and I'll update the PR accordingly.

**Comment by alessio** [comment]
The full diff is available here: https://patch-diff.githubusercontent.com/raw/cosmos/cosmos-sdk/pull/6725.diff

File: go.mod [hunk 2]
```diff
@@ -6,7 +6,7 @@ require (
...
+	github.com/cosmos/cosmos-sdk v0.39.0-rc0
```
Comment (alessio) [inline go.mod:9]: @jackzampolin there it is

**Comment by alessio** [comment]
@boz dixit:

> Question: can we use tendermint's v3 blockchain reactor with this?

AFAIK the `v3` reactor is not a thing, though I suspect you were referring to the `v2` reactor :)
In that case the answer is yes, you can use it at your own risk. The Tendermint Core folks haven't yet tested it fully.

Hope this helps!

**Comment by boz** [comment]
Great, thanks!  Yes, meant `v2`.

**Pull request closed without merging**